src/Native/Common/pal_config.h.in

```diff
@@ -36,6 +36,9 @@
 #cmakedefine01 HAVE_TCP_VAR_H
 #cmakedefine01 HAVE_RT_MSGHDR
 #cmakedefine01 HAVE_LINUX_RTNETLINK_H
+#cmakedefine01 HAVE_INOTIFY_INIT
```

Comment:
@pgavlin Agreed that it is, however some distros have ports of Linux-specific APIs that would fail based on this check..for example, *BSD distros have an inotify-compatible port here

http://www.freshports.org/devel/libinotify

If we just check for the compatible functions, then this scenario will work on platforms with this port enabled (in theory) :)